### PR TITLE
[docs] Added recording for 202405 meeting

### PR DESCRIPTION
**Pull request by andrewnicols**

### CODE REVIEW

**Review by timhunt — APPROVED** [review]
Thanks Andrew. Looks good to me.

File: general/community/meetings/2024/05.md [hunk 1]
```diff
@@ -10,11 +10,11 @@ sidebar_position: -05
 
 24th May 2024 - Time 7:00AM UTC
 
-[BBB Meeting link](https://moodle.org/mod/bigbluebuttonbn/view.php?id=8596)
+[Meeting Recording](https://moodle.org/mod/bigbluebuttonbn/bbb_view.php?action=play&bn=1&rid=49&rtype=video)
 
 ## Agenda
 
-1. Moodle Developer Experience (DX) update
+1. [Moodle Developer Experience (DX) update](./_files/2024-03-Mikel-presentation.pdf)
```
Comment (sarjona) [inline general/community/meetings/2024/05.md:17]:
```suggestion
1. [Moodle Developer Experience (DX) update](./_files/2024-05-DX-presentation.pdf)
```

Comment (sarjona) [reply general/community/meetings/2024/05.md:17]: This should point to the uploaded file, right?

File: general/community/meetings/2024/05.md
Comment (sarjona) [inline general/community/meetings/2024/05.md:1]: Apart from updating the presentation link, shouldn't the May meeting be listed in the general/community/meetings page too?

**Review by sarjona — CHANGES_REQUESTED** [review]
Hi @andrewnicols!
I left a couple of comments but I've just seen you just merged it... Feel free to create a separate issue to fix them ;-)

**Comment by timhunt** [comment]
Sorry, @sarjona. Mid-air collision.

**Comment by sarjona** [comment]
No need to apologise, @timhunt!
They are easy to fix in a separate issue ;-)

**Pull request merged**